package-lock.json

```diff
@@ -0,0 +1,7617 @@
+{
```

Comment:
I understand this file has been missadded (a "git add" too broad? ;) and should be removed from the PR.

Reply:
possible :)

Reply:
fixed by f7e4b6d45cde9cc0a385b317ffb3536539f64a67